src/assets/user-info-settings/index.css

```
@@ -0,0 +1,298 @@
+	float: left;
+	width: 200px;
+	box-sizing: content-box;
+	-webkit-box-sizing: content-box;
+	-webkit-box-align: center;
+	-webkit-box-pack: center;
+}
+
+.account-info > img {
+	display: inline-block;
+	width: 128px;
+	height: 128px;
+	overflow: hidden;
+	border-radius: 100%;
+	background: #E4DBF2;
+	margin: 30px 30px 0 35px;
+	box-shadow: 0-1px 2px rgba(0, 0, 0, 0.4);
+}
+
+.account-info > .content {
+	text-align: center;
+	color: #673DB6;
+	margin: 0;
+}
+
+.account-info > h3.content {
+	font-weight: normal;
+	color: #9D6FF3;
+	font-size: 20px;
+}
+
+.account-info .access-level-banned {
+	background: rgba(255, 63, 0, 0.3);
+	color: #FF3F00;
+	border-color: #FF3F00;
+}
+
+.account-info .tier-level-1 {
+	background: #934D4D;
+	color: #FF8484;
+	border-color: #FF8484;
+}
+
+.account-info .tier-level-2 {
+	background: #316C59;
+	color: #59C9A5;
+	border-color: #59C9A5;
+}
+
+.account-info .tier-level-3 {
+	background: #6B5E84;
+	color: #CAB1FB;
+	border-color: #CAB1FB;
+}
+
+.account-info .access-level-1 {
+	background: #3B918C;
+	color: #64F7EF;
+	border-color: #64F7EF;
+}
+
+.account-info .access-level-2 {
+	background: #917235;
+	color: #FFC759;
+	border-color: #FFC759;
+}
+
+.account-info .access-level-3 {
+	background: #3A973C;
+	color: #5AFF15;
+	border-color: #5AFF15;
+}
+
+.account-info .tier-name {
+	margin: 12px auto;
+	line-height: 1.2em;
+	border-radius: 1.2em;
+	border-width: 2px;
+	border-style: solid;
+	padding: 4px 16px;
+	width: min-content;
+	display: inline-block;
+}
+
+div.group > p.content > button {
+	min-width: 100px;
+	min-height: 20px;
+	display: block;
+	width: 100px;
+	float: right;
+}
+
+div.radio {
+	display: inline-block;
+}
+
+.server-selection input {
+	opacity: 0;
+	position: absolute;
+	width: 200px;
+	height: 70px;
+	cursor: pointer;
+}
+
+div.radio label {
+	display: inline-block;
+	color: #6A6C75;
+	min-width: 200px;
+	box-sizing: content-box;
+	-webkit-box-sizing: content-box;
+	-webkit-box-align: center;
+	-webkit-box-pack: center;
+	text-align: center;
+	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), color-stop(0.8, #F6F6F6), color-stop(0.95, #F5F5F5), to(#BBB))0 0;
+	border-radius: 12px;
+	cursor: pointer;
+	border: 0;
+	box-shadow: 0 1px 5px rgba(0, 0, 0, 0.4);
+	margin: 5px;
+	font-weight: normal;
+	font-size: 15px;
+}
+
+div.radio label h2 {
+	display: inline-block;
+	font-weight: normal;
+}
+
+div.radio label svg {
+	margin: -6px 0;
+}
+
+header .fixed-bottom-button.left {
+	padding: 0 60px 0 40px !important;
+}
+
+.fixed-bottom-button.left {
+	right: auto;
+	left: 0;
+	border-radius: 0 40px 0 0;
+}
+
+header .fixed-bottom-button {
+	min-width: 120px;
+	padding: 0 40px 0 60px;
+}
+
+.fixed-bottom-button,
+input[type="submit"] {
+	position: fixed;
+	bottom: 0;
+	right: 0;
+	height: 85px;
+	line-height: 100px;
+	padding: 0 40px 0 60px;
+	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), to(#E6E6E6))0 0;
```

Nitpick. Missing space

```suggestion
	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), to(#E6E6E6)) 0 0;
```